File: 3oy1 Muhriddin Sobirjonov/8masala.c
#include    <stdio.h>
#include    <stdlib.h>
#include    <time.h>
#include    <math.h>

void    TOLDIRISH(int array[], int size){
    srand(time(0));
    for(int i = 0; i < size; i++){
        array[i] =rand()%60 +2;
        printf("%d\t", array[i]);
    }
}
void    TUB(int *arr, int size, int array[]){
    for(int i = 0; i < size; i++){
        int yig = 0;
        for(int k = 1; k < sqrt(array[i]); k++ ){
            if(*(arr + i ) % k == 0){
                yig ++;
            }
        }
        if(yig == 1){
            printf(" %d\t", *(arr + i) );
        }
    }
}

int main(){
    system("cls");
    int array[10];
    int size = sizeof(array) / sizeof(int);
    TOLDIRISH(array, size);
    puts("\n");
    printf("Tub sonlar:  ");
    TUB(array, size, array);

    return 0;
}

File: 4oy3 Muhriddin Sobirjonov/8masala.py
import os
os.system("cls")

n = int (input("N:  "))
yig = 0
son = 2
for i in range(1, n + 1):
    if i != n:
        print(son, end=" + ")
    else:
        print(son, end=" = ")
    yig += son
    son = son * 10 + 2
print(yig)
    
    






